src/main/java/com/bandwidth/sqs/publisher/MessagePublisher.java

```diff
@@ -13,22 +10,20 @@
     /**
      * Publishes a message immediately, with the given delay
      *
-     * @param message The message to publish
-     * @param queueUrl The queue to send the message to
+     * @param body The message body to publish
      * @param maybeDelay Amount of time a message is delayed before it can be consumed (Max 15 minutes)
      *                   or the default delay of the SQS queue if "empty"
      * @return The message id
      */
-    Single<String> publishMessage(T message, String queueUrl, Optional<Duration> maybeDelay);
+    Single<String> publishMessage(T body, Optional<Duration> maybeDelay);
```

Comment:
Not really sure what needs documented here. This is an interface that the user won't directly create.
The javadoc already lists all implementing classes of interfaces so it's easy to see how to get a `MessagePublisher`

see example here: http://dev.bandwidth.com/async-sqs/apidocs/com/bandwidth/sqs/publisher/MessagePublisher.html